webclient/src/views/view/view-view.inc

```diff
@@ -8,6 +8,32 @@
          class="img-responsive"/>
   </a>
 
+  <!-- Pending coordinates counter (if any) -->
+  <div class="panel coord-counter" v-if="coord_counter.counter">
+    <div class="panel-body columns">
+      <div class="column col col-sm-12 msg">
+        ${{_.view_view.msg.coordinate-counter.msg-1}}$
+        <em>{{ coord_counter.counter }}</em>
+        ${{_.view_view.msg.coordinate-counter.msg-2}}$
+        <button class="btn btn-action btn-sm btn-link tooltip tooltip-left"
+                data-tooltip=" ${{_.view_view.msg.coordinate-counter.info}}$ ">&#x1f6c8;</button>
+      </div>
+      <div class="column col-auto col-sm-12 btns">
+        <button class="btn btn-link btn-sm delete"
+                v-bind:class="{'to-confirm': coord_counter.to_confirm_delete}"
+                @click="deletePendingCoordinates">
+          <i class="icon icon-cross"></i>
+          <span class="default">${{_.view_view.msg.coordinate-counter.delete}}$</span>
+          <span class="confirm">${{_.view_view.msg.coordinate-counter.delete-confirm}}$</span>
+        </button>
+        <button class="btn btn-primary btn-sm" @click="openFeedbackForm">
+          <i class="icon icon-check"></i>
+          ${{_.view_view.msg.coordinate-counter.send}}$
+        </button>
```

Comment:
It's probably not obvious, but you can always open the feedback button again and edit. Also the 'inaccurate coordinate' warning doesn't disappear, so you can click this again to edit.

So if possible I would like to avoid a third button here (because it also takes more space on mobile). But if we feel the above is not clear enough there are maybe ways to make that clearer.